spec/cache_crispies/hash_builder_spec.rb

```diff
@@ -24,10 +24,19 @@ def small_print
     end
   end
 
+  class AddressSerializer < CacheCrispies::Base
+    serialize :full_address do |model, _options|
+      "#{model.street}, #{model.city}, #{model.state} #{model.zip}"
+    end
+  end
+
   class CerealSerializerForHashBuilder < CacheCrispies::Base
     serialize :uid, from: :id, to: String
     serialize :name, :company
     merge :itself, with: MarketingBsSerializer
+    merge :itself, with: AddressSerializer do |model, _options|
+      model.address
+    end
```

Comment:
Could you explain the advantage of doing this over just using the attribute directly as described below?
```ruby
merge :address, with: AddressSerializer
```